modules/core/history.js

```diff
@@ -281,9 +281,12 @@ export function coreHistory(context) {
 
 
         validate: function(changes) {
-            return _flatten(_map(Validations, function(fn) {
-                return fn(context)(changes, _stack[_index].graph);
-            }));
+            return _flatten(_map(
+                _filter(Validations, _isFunction),
```

Comment:
i think this extra check should probably happen just once when the `Validations` is created